extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json

```diff
@@ -104,6 +104,27 @@
           },
           "meta:titleId": "message-delivery-feedback##https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/messageFailure##title##88741"
         },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/source": {
+          "title": "Message Source",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/sourceType": {
+              "title": "Source Type",
```

Comment:
@anshchauhan do you think this should be an enumeration or should we leave this as a freeform string?

CC: @ankusharma246

Reply:
+1. This should be enum.